### ISSUE TITLE
[Feature Request] Centralised crafting with Xnet channel or send crafting requests over network message

### ISSUE THREAD

**Issue by almightyju** [McJty/RFToolsControl#174]
I would like a way to have a central crafting controller (much like AE2 and RS but using RFTools), If a crafter could be linked to multiple processors via XNet cables rather than the range of network cards this would be possible.

As a work around I tried to create a processor that on event: craft would send that via a message to a remote processor for the remote processor to craft and then place in storage and send a message back with the crafting result. This would allow me to have a room with a single crafter and lots of processors which all relay messages to other processors, sadly this didn't work. 

It doesn't seem possible to store a crafting ticket into a variable let alone send it as a message, the closest I found to getting the crafting ticket was the log with function:ticket but this seems to log the block co-ords for the crafting station with a :X (not sure what X represents), but still doesn't allow me to do anything else, storing a variable doesn't have the ticket as an option and using last_int doesn't work, the remote processor is unable to get the crafting ticket remotely.

So in short an XNet channel that connects crafters to programmers (maybe just using the storage channel would work as a global "RFTools" channel) would create a great central crafting control, or having a way to transmit crafting tickets to remote processors would be a good backup option.
